modules/cluster-logging-collector-log-forwarding-supported-plugins-5-2.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-log-forwarding-supported-plugins-5-2_{context}"]
+
+= Supported log data output types in OpenShift Logging 5.2
+
+Red Hat OpenShift Logging version 5.2 provides the following output types and protocols for sending log data to target log collectors.
+
+Red Hat tests each of the combinations shown in the following table. However, you should be able to send log data to a wider range target log collectors that ingest these protocols.
+
+[options="header"]
+|====
+| Output types   | Protocols          | Tested with
+
+| Amazon CloudWatch
+| TBD
+| TBD
+
+
+| elasticsearch
+| elasticsearch
+a| Elasticsearch 6.8.1
+
+Elasticsearch 6.8.4
+
+Elasticsearch 7.12.2
+
+| fluentdForward
+| fluentd forward v1
+a| fluentd 1.7.4
+
+logstash 7.10.1
+
+| Grafana Loki
```

Comment:
@alanconway @kabirbhartiRH Would you help me describe the protocol and version we test for this output?

Reply:
@alanconway @kabirbhartiRH It's unclear which information from https://github.com/openshift/origin-aggregated-logging/blob/release-5.2/fluentd/Gemfile.lock belongs here.

Reply:
@rolfedh the Gemlock versions don't  belong here. They describe what gems (libraries) we use _inside_ the forwarder.  This section is describing product versions that have tested forwarding to _outside_ the forwarder. Those versions don't correspond. Even in the case of fluetnd itself, we may be using an older version of fluentd but have tested with a newer (but wire-compatible) version for external forwarding (or vice-versa)